nav2_gps_waypoint_follower_demo/launch/dual_ekf_navsat.launch.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
```

Comment:
What's this `nav2_gps_waypoint_follower_demo/launch/__pycache__/gps_waypoint_follower.launch.cpython-310.pyc` file that's 2KB added?

Reply:
removed, my bad